### PR TITLE
Session version

### PR DESCRIPTION
**Pull request by Dire-0x**
Notify user of new app version

### CODE REVIEW

**Comment by donvincenz0** [comment]
LG

File: src/constants/index.ts [hunk 1]
```diff
@@ -2,6 +2,8 @@ import { ChainId, Exchange, JSBI, Percent, Token, WETH } from '@alchemistcoin/sd
 import { AbstractConnector } from '@web3-react/abstract-connector'
 import { injected, ledger, walletconnect } from '../connectors'
 
+export const APP_VERSION = '0.1.5'
```
Comment (juice0x) [inline src/constants/index.ts]: We should be able to import the version directly from package.json so it doesn't have to be manually updated here. Security concerns are minimal as the package.json is already exposed publicly

**Review by donvincenz0 — APPROVED** [review]
LG

**Review by juice0x — APPROVED** [review]
lgtm

File: src/constants/index.ts (hunk 1 above)
Comment (Dire-0x) [reply src/constants/index.ts]: Yea good call

**Pull request merged**